packages/logger/README.md

```diff
@@ -97,17 +98,20 @@ The library exposes the following methods:
 | `.createLogger` | Creates the logger object which can be used |
 | `.createFluentTransport` | Creates a FluentD compatible transport |
 | `.createConsoleTransport` | Creates a Console transport |
+| `.createFileTransport` | Creates a File transport |
 | `.createMorganStream` | Creates an object consumable by Morgan |
 
 ### `.createLogger(:options)`
 This function accepts a configuration object as the parameter where the keys are documented as follows:
 
 | Key | Description | Default |
 | --- | --- | --- |
+| `additionalTransports` | Winston transports to be added on (no overriding of the default Console transport) | `[]` |
 | `formatters` | Winston formatters created via `winston.format(...)()` | `[]` |
 | `level` | Default level (`ENUM { "error", "warn", "info", "http", "debug", "silly" }`) | `"silly"` |
+| `namespace` | Give a name to your logger) | `give-me-a-name` |
```

Comment:
This was from previous MR !! Anyway it's just displaying a label in the logs so that you know which logger is logging. 

In the end the idea is to have something like debug library (being able to enable/disable logs depending on env vars). We will see if we can make it :)

Reply:
yeah i saw the label after running the example, not sure how it'll turn up in kibana but let's see how that goes